composer.json

```diff
@@ -26,22 +26,19 @@
         "friends-of-behat/symfony-extension": "^2.1",
         "friends-of-behat/variadic-extension": "^1.3",
         "lchrusciel/api-test-case": "^4.1||^5.0",
-        "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev",
         "phpspec/phpspec": "^7.0",
         "phpstan/extension-installer": "^1.0",
         "phpstan/phpstan": "0.12.90",
         "phpstan/phpstan-doctrine": "0.12.39",
         "phpstan/phpstan-strict-rules": "^0.12.0",
         "phpstan/phpstan-webmozart-assert": "0.12.12",
         "phpunit/phpunit": "^9.5",
-        "sensiolabs/security-checker": "^6.0",
         "sylius-labs/coding-standard": "^4.0",
         "symfony/browser-kit": "^4.4 || ^5.2",
         "symfony/debug-bundle": "^4.4 || ^5.2",
         "symfony/dotenv": "^4.4 || ^5.2",
         "symfony/intl": "^4.4 || ^5.2",
```

Comment:
every single symfony package should be "^5.4 || ^6.0" - see sylius plugin skeleton composer - https://github.com/Sylius/PluginSkeleton/blob/1.12/composer.json